package.json

```diff
@@ -38,18 +38,15 @@
   },
   "lint-staged": {
     "**/*": [
-      "suppress-exit-code eslint --fix",
```

Comment:
Not sure how to run workspace-specific ESLint in a pre-commit hook, but I hope we can live without autofixes. ESLint should apply on save in a correctly configured IDE.